lib/options/uefi-setup-menu.robot

```diff
@@ -59,3 +60,67 @@ Get UEFI Option
 
     ${state}=    Get Option State    ${menu}    ${option_path[${path_len}-1]}    ${VALUE}
     RETURN    ${state}
+
+
+Get Boot Time From Cbmem
+    [Documentation]    Calculates boot time based on cbmem timestamps
+    # fix for LT1000 and protectli platforms (output without tabs)
+    Get Cbmem From Cloud
+    ${out_cbmem}=    Execute Command In Terminal    cbmem -T
+    Should Not Contain
+    ...    ${out_cbmem}
+    ...    Operation not permitted
+    ...    msg=Cannot get cbmem log. Probably Secure Boot is enabled (kernel lockdown mode).
+    ${lines}=    Split To Lines    ${out_cbmem}
+    ${first_line}=    Get From List    ${lines}    0
+    ${last_line}=    Get From List    ${lines}    -1
+    ${first_timestamp}=    Get Timestamp From Cbmem Log    ${first_line}
+    ${last_timestamp}=    Get Timestamp From Cbmem Log    ${last_line}
+    ${boot_time}=    Evaluate    (${last_timestamp} - ${first_timestamp}) / 1000000
+    RETURN    ${boot_time}
+
+Get Timestamp From Cbmem Log
```

Comment:
These are the same between dcu.robot and uefi-setup-menu.robot, right? Please deduplicate and move to another shared file

Reply:
Done